src/d/d_menu_collect.cpp

```diff
@@ -5,15 +5,328 @@
 
 #include "d/d_menu_collect.h"
 #include "dolphin/types.h"
+#include "stdio.h"
+#include "d/d_com_inf_game.h"
+
+#include "weak_bss_936_to_1036.h" // IWYU pragma: keep
+#include "weak_data_1811.h" // IWYU pragma: keep
+
+static dMc_HIO_c g_mcHIO;
 
 /* 8019BE8C-8019BF14       .text __ct__9dMc_HIO_cFv */
 dMc_HIO_c::dMc_HIO_c() {
-    /* Nonmatching */
+    /* Fakematch */
+    m58 = -1;
+    m46 = 40;
+    m48 = 10;
+    m4A = 120;
+    m4C = 100;
+    m54 = 4.0f;
+    m5C = 0xff;
+    m5D = 70;
+    m52 = 130;
+    m4E = 250;
+    m50 = 20;
+    ((u8*)&m58)[0] = -1;
+    ((u8*)&m58)[1] = 170;
+    ((u8*)&m58)[2] = 0;
+    ((u8*)&m58)[3] = 0;
```

Comment:
Maybe this is a `JUtility::TColor`? Default ctor sets it's to a 32-bit `0xFFFFFFFF`, but normally it's 4 separate 8-bit values.